pkg/maps/lbmap/lbmap.go

```diff
@@ -31,9 +32,9 @@ import (
 
 var log = logging.DefaultLogger.WithField(logfields.LogSubsys, "map-lb")
 
-const (
-	// Maximum number of entries in each hashtable
-	MaxEntries = 65536
+var (
+	// MaxEntries is the maximum number of entries in each hashtable
+	MaxEntries = defaults.LBMapEntries
```

Comment:
Not sure whether it makes sense to have another indirection by storing the actual number in `defaults`. I was suggesting to have it here.

Reply:
Yeah, I think it makes sense. I will update the PR.